### ISSUE TITLE
Provide config with sensible defaults for Dosbox

### ISSUE THREAD

**Issue by strycore** [lutris/lutris#326]
So far, I've been told we should set autolock to False, which I agree with. What else should we set in the default config?

**Comment by RobLoach** [comment]
I personally like `autolock`. It's great for games that use the mouse, like Abuse, Simcity, etc.

**Comment by strycore** [comment]
is there a way to set this as a runner option? Like Wine does?

**Comment by Xodetaetl** [comment]
> is there a way to set this as a runner option? Like Wine does?

Yes, by creating a config file on the fly and appending it to the command. It's the only way to ensure that options set from Lutris will override existing config.

**Comment by strycore** [comment]
The defaults are ok, the only problem left is with with fullresolution which should be set to 2560x1440 or whatever the current resolution is.

**Comment by 7HEPOW** [comment]
Just a small comment on resolutions and mouselock:

Mouselock:
The issue has been fixed in the SVN version as of 4177. [SVN](https://sourceforge.net/p/dosbox/code-0/4177/) / [Forum thread on page 4](https://www.vogons.org/viewtopic.php?f=31&t=62703)
In 0.74.2 this is still an issue, but it can be patched with the .diff file posted on Decenber 12th, 2018 over on the main page at: https://www.dosbox.com/
Just in case somebody wants to provide a patched 0.74.2 version to download from inside Lutris directly. :wink: 

Fullscreen:
I guess the good old multi-monitor setups will behave a little weird if no resolution is passed to dosbox. I'm on a dual 1080p (XFCE) setup and dosbox likes to center the displayed image across both screens when told to launch in fullscreen with no resolution given. When telling it to use 1920x1080 it launches perfectly fine on the screen of **the currently active window** it's launched from. In my case: Lutris on the left screen launches dosbox on the left screen and Lutris on the right screen does the same on the right screen in both windowed or fullscreen.
So in this case, when setting any game to launch in fullscreen, it would be best to have Lutris fetch the resolution of the screen it's on and pass that to dosbox. This would also prevent resolution changes made to the desktop which will stick, when dosbox crashes for any reason. :smiley: 
Creating a config file on the fly to pass on launch as @Xodetaetl metioned would only need this in it (assuming we also want to autolock on fullscreen):
```
[sdl]
fullscreen=true
fullresolution=1920x1080
autolock=true
```
Only these settings will then be overridden from any previously loaded config file.
